public/js/index.js

```diff
@@ -116,9 +116,15 @@ $(function () {
     $('.widget').each((i, element) => {
       objSaveConfig.widgets.push($(element).data('widget'))
     })
-    $.post('/config', objSaveConfig, function (objResponse) {
-      console.log('Save Config Response', objResponse)
-    })
+    $.ajax({
+      type: "POST",
+      url: '/config',
+      contentType: "application/json",
+      data: JSON.stringify(objSaveConfig),
+      success: function (objResponse) {
```

Comment:
The backend includes a failure reason with the 400 status code. That should be displayed somewhere, such as the console.log.

Reply:
error added